File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/ruby/practicas_3/a_que_equipo_le_das.rb

puts "Cuantas personas van a participar en la encuesta:"
cuantos = gets.chomp.to_i

for vueltas in 1..cuantos

    puts "A que equipo le das:"
    team = gets.chomp

    puts "Porque razon:"
    why = gets.chomp

    puts "----------------------------------------------------------"

    puts "ENCUESTADO N #{vueltas}:"

    puts "----------------------------------------------------------"

    puts "Equipo favorito: #{team}"
    puts "Porque: #{why}"

    puts "----------------------------------------------------------"
    
end

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/dart/practicas_3/a_que_equipo_le_das.dart
import 'dart:io';

void main () {

  print('Cuantas personas van a participar en la encuesta:');
  int cuantos = int.parse(stdin.readLineSync()!);

  for (int vueltas = 1; vueltas <= cuantos; vueltas++) {

    print('A que equipo le das:');
    String team = stdin.readLineSync()!;

    print('Porque razon:');
    String why = stdin.readLineSync()!;

    print('----------------------------------------------------------');

    print('ENCUESTADO N $vueltas:');

    print('----------------------------------------------------------');

    print('Equipo favorito: $team');
    print('Porque: $why');

    print('----------------------------------------------------------');

  }

}